### PR TITLE
Fixed git-remote-gosh clone operation

### PR DESCRIPTION
**Pull request by SilkovAlexander**

### CODE REVIEW

File: v1_x/git-remote-gosh/src/blockchain/tag/save.rs [hunk 1]
```diff
@@ -61,7 +64,29 @@ impl Tagging for Everscale {
...
+            Some(serde_json::json!({ "_nametag": tag_name })),
+        )
+        .await?;
+
+        let expected_tag_contract = GoshContract::new(&address, gosh_abi::TAG);
+        if expected_tag_contract.is_active(&client).await? {
+            anyhow::bail!("hint: Updates were rejected because the tag already exists in the remote.");
+        }
+
+        let wallet_contract = wallet.take_one().await?;
```
Comment (sauin) [inline v1_x/git-remote-gosh/src/blockchain/tag/save.rs]: Tag should be deployed via zero wallet

File: v2_x/git-remote-gosh/src/blockchain/tag/save.rs [hunk 2]
```diff
@@ -61,7 +64,29 @@ impl Tagging for Everscale {
...
+            Some(serde_json::json!({ "_nametag": tag_name })),
+        )
+        .await?;
+
+        let expected_tag_contract = GoshContract::new(&address, gosh_abi::TAG);
+        if expected_tag_contract.is_active(&client).await? {
+            anyhow::bail!("hint: Updates were rejected because the tag already exists in the remote.");
+        }
+
+        let wallet_contract = wallet.take_one().await?;
```
Comment (sauin) [inline v2_x/git-remote-gosh/src/blockchain/tag/save.rs]: Same as in the v1, tag should be deployed via zero wallet

**Review by sauin — CHANGES_REQUESTED** [review]

File: v1_x/git-remote-gosh/src/blockchain/tag/save.rs (hunk 1 above)
Comment (SilkovAlexander) [reply v1_x/git-remote-gosh/src/blockchain/tag/save.rs]: Fixed

File: v2_x/git-remote-gosh/src/blockchain/tag/save.rs (hunk 2 above)
Comment (SilkovAlexander) [reply v2_x/git-remote-gosh/src/blockchain/tag/save.rs]: Fixed

**Review by sauin — APPROVED** [review]

**Pull request merged**